src/lib/components/Speck.react.js

```diff
@@ -8,70 +8,11 @@ import {
     speckPresetViews,
 } from 'speck';
 
-export default class Speck extends Component {
```

Comment:
We use this structure in our other libraries (e.g., https://github.com/plotly/dash-core-components/blob/master/src/components/Input.react.js). I think it's OK to keep it for now if it's not affecting the performance or functionality in any way. If this is a convention that is widely used, however, it would be worthwhile to look into implementing it everywhere (but that's a discussion with a larger scope than just this component). 

cc @alexcjohnson

Reply:
In light of the above comment, could you please change this back to the way it was, so that the diffs are clearer?

Reply:
I only chose to do it this way because the linter wants me to keep `React` in scope and keeping that in mind using `React.Component` results in the least amount of code lol. I can change it back.